terra_layer/models.py

```diff
@@ -80,7 +92,7 @@ def save(self, **kwargs):
 
         # Invalidate cache for layer group
         if self.group:
-            cache.delete(get_layer_group_cache_key(self.group.view))
+            cache.delete(get_layer_group_cache_key(self.group.view.pk))
```

Comment:
I think here, get_layer_group_cache_key should be modified, instead of giving the pk of the view, so if we decide to change the way to create the cache key we have access to whole view object.

Reply:
Done